src/vrapp/VRVolumeApp.cpp

```diff
@@ -901,6 +903,7 @@ void VRVolumeApp::clear_data()
 		}
 	}
 	m_volumes.clear();
+	m_numVolumes = 0;
```

Comment:
I think this may be an used variable?

Reply:
Not really. That variable is needed by other methods. `void VRVolumeApp::init_num_volumes(int nVolumes)`.